src/www/widgets/widgets/ipsec.widget.php

```diff
@@ -77,29 +77,46 @@
 ?>
 <script>
     $(document).ready(function() {
-        $(".ipsec-tab").unbind('click').click(function(){
-            $(".ipsec-tab").css('background-color', '#777777');
-            $(".ipsec-tab").css('color', 'white');
-            $(this).css('background-color', '#EEEEEE');
-            $(this).css('color', 'black');
+        $(".ipsec-tab").unbind('click').click(function() {
```

Comment:
@AdSchellevis should we remove unbind too - it has been removed from standard jquery?

Reply:
https://github.com/opnsense/core/pull/3286#issuecomment-469782949 , but I don't mind.